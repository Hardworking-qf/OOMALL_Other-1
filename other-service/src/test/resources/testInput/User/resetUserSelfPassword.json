{
  "1": {
    "userName": "XQChen",
    "mobile": "12300010003",
    "email": "chenxiaoquan233@gmail.com"
  },
  "2": {
    "userName": "XQChen",
    "mobile": "12300010003",
    "email": "chenxiaoquan23343@gmail.com"
  },
  "3": {
    "userName": "XQChen2",
    "mobile": "12300010003",
    "email": "chenxiaoquan233@gmail.com"
  }
}