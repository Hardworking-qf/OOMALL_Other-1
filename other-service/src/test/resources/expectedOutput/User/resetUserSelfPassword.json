{
  "1": {
    "errno": 0,
    "errmsg": "成功"
  },
  "2": {
    "errno": 746,
    "errmsg": "与系统预留的电话不一致"
  },
  "3": {
    "errno": 745,
    "errmsg": "与系统预留的邮箱不一致"
  },
  "4": {
    "errno": 700,
    "errmsg": "用户名不存在或者密码错误"
  }
}